synapse/models/inet.py

```diff
@@ -3234,6 +3244,38 @@ def getModelDefs(self):
                             'doc': 'The server that was sampled to compute the JARM hash.'}),
                     )),
 
+                    ('inet:ssl:ja3hash', {}, (
+                        ('version', ('int', {}), {
```

Comment:
So the JA3 spec pulls/needs these values as the decimal values pulled from the PCAP files. 

Most data sources I've seen save for like a raw pcap only give the hash and never even give these, so these are kind of moot point unless you've got access to a raw PCAP.

But for completeness's sake, do we want to provide lookup values for each of these, since otherwise people will have to dig through a bunch of RFC's to know what they mean (or is that a bit too much for now?)